config/locales/en.yml

```diff
@@ -150,6 +162,7 @@ en:
     logs: "Logs"
     no_steps: "No timeline steps to display."
     project_step: "project step"
+    project_steps: "project steps"
```

Comment:
The Rails way to do plurals like this is to do
```
project_step:
  one: "project step"
  other: "project steps"
```

Then when you translate you do e.g. `t("project_step", count: 2)`